src/App.tsx

```diff
@@ -1,8 +1,97 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { getTodos, getUsers } from './helper/api';
+import { SortPanel } from './components/SortPanel';
+import { TodoList } from './components/TodoList';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+const normalizeTodos = async () => {
+  const todos = await getTodos();
+  const users = await getUsers();
+
+  return todos.map((todo: Todo) => {
+    return {
+      ...todo,
+      user: users.find((user: User) => user.id === todo.userId),
+    };
+  });
+};
+
+const App = () => {
+  const [loading, setLoading] = useState(false);
+  const [visibleContent, setVisibleContent] = useState(false);
```

Comment:
```suggestion
  const [isContentVisible, setContentVisibility] = useState(false);
```
don't forget it's a boolean value